actors/user/src/main/java/org/sunbird/user/service/impl/VerificationServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package org.sunbird.user.service.impl;
+
+import java.util.Map;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.user.service.GoogleAuthCodeVerificationService;
+import org.sunbird.user.service.VerificationService;
+
+public class VerificationServiceImpl implements VerificationService {
```

Comment:
Convert this to factory - class - giving instance of Verifier - based on supported source Types.